project/Dependencies.scala

```diff
@@ -1,34 +1,89 @@
+      "com.typesafe.akka" %% "akka-actor"                         % AkkaVersion
+    )
+
+    val AkkaHttpCore = Seq(
+      "com.typesafe.akka" %% "akka-http-core"                     % AkkaHttpVersion,
+      "com.typesafe.akka" %% "akka-http-spray-json"               % AkkaHttpVersion,
+      "io.spray"          %% "spray-json"                         % SprayJsonVersion                  // ApacheV2
+    )
+    val AkkaHttp = Seq(
       "com.typesafe.akka" %% "akka-http"                          % AkkaHttpVersion,
       "com.typesafe.akka" %% "akka-http-spray-json"               % AkkaHttpVersion,
-      "io.spray"          %% "spray-json"                         % SprayJsonVersion,                  // ApacheV2
+      "com.typesafe.akka" %% "akka-stream"                        % AkkaVersion,
+      "com.typesafe.akka" %% "akka-actor"                         % AkkaVersion,
+      "io.spray"          %% "spray-json"                         % SprayJsonVersion                  // ApacheV2
+    )
 
-      // TODO this would be needed to use the SRV records, but more likely we want to implement it ourselves
-      // "ru.smslv.akka"     %% "akka-dns"                           % akkaDns, // ApacheV2
+    val AkkaSharding = Seq(
+      "com.typesafe.akka" %% "akka-cluster"                       % AkkaVersion,
+      "com.typesafe.akka" %% "akka-cluster-sharding"              % AkkaVersion
+    )
 
-      "com.typesafe.akka" %% "akka-distributed-data"              % AkkaVersion     % "test",
+    val AkkaTesting = Seq(
+      "com.typesafe.akka" %% "akka-testkit"                       % AkkaVersion     % "test",
       "com.typesafe.akka" %% "akka-http-testkit"                  % AkkaHttpVersion % "test",
       "junit"             % "junit"                               % JUnitVersion    % "test",
       "org.mockito"       % "mockito-all"                         % "1.10.19"       % "test"  // Common Public License 1.0
     )
+  }
+
+  val Discovery = Seq(
+    libraryDependencies ++=
+      DependencyGroups.AkkaActor ++
+      DependencyGroups.AkkaTesting
+  )
+
+  val DiscoveryDns = Seq(
+    libraryDependencies ++=
+      DependencyGroups.AkkaActor ++
+      DependencyGroups.AkkaTesting ++ Seq(
+        // FIXME this would be needed to use the SRV records, but more likely we want to implement it ourselves
+        // "ru.smslv.akka"     %% "akka-dns"                           % akkaDns, // ApacheV2 (we do need it)
+      )
+  )
+
+  val ManagementHttp = Seq(
+    libraryDependencies ++=
+      DependencyGroups.AkkaHttp ++
+      DependencyGroups.AkkaTesting
+  )
+  val ClusterHttp = Seq(
+    libraryDependencies ++=
+      DependencyGroups.AkkaSharding ++
+      DependencyGroups.AkkaHttpCore ++
+      DependencyGroups.AkkaTesting ++ Seq(
+      "com.typesafe.akka" %% "akka-distributed-data"              % AkkaVersion     % "test"
+    )
+  )
+
+  val ClusterBootstrap = Seq(
+    libraryDependencies ++=
+      DependencyGroups.AkkaSharding ++
```

Comment:
Can we get away without `AkkaSharding` for `ClusterBootstrap`? Don't we just need `akka-cluster`?

Reply:
Good catch, thanks! Yeah only cluster is needed

Reply:
Still have sharding here